src/Microsoft.ServiceFabric.Services/ServiceTelemetry.cs

```diff
@@ -0,0 +1,187 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.ServiceFabric.Services
+{
+    using System.Fabric;
+
+    /// <summary>
+    /// ServiceTelemetry contains the telemetry methods for ServiceFramework.
+    /// </summary>
```

Comment:
Add in the summary that this class is to be used by other Service Fabric  components only.

Reply:
Thanks, I have added that to the summary.